main.go

```diff
@@ -143,3 +156,43 @@ func printVersion() {
 	setupLog.Info(fmt.Sprintf("Git Commit: %s", version.GitCommit))
 	setupLog.Info(fmt.Sprintf("Build Date: %s", version.BuildDate))
 }
+
+func printSupportedFenceAgents(agentList []string) {
+	setupLog.Info(fmt.Sprintf("Fence agents: %d", len(agentList)))
+	setupLog.Info(fmt.Sprintf("Available agents: %s", strings.Join(agentList, " ")))
+}
+
+// checkFenceAgentsFileExists check if the file exists, read it, and then remove redundant prefix
+func checkFenceAgentsFileExists(filePath string) ([]string, error) {
```

Comment:
I think method name should be changed - it mainly extract the fence agents from the file.
IMO `getFenceAgents` (or something similar)  is a better name.

Another option is to break this method into two methods (one that checks for the file and one that retrieves the file)

I'm leaning into the first option of just changing the name but I think both options are fine.